sentry/src/main/java/io/sentry/MainEventProcessor.java

```diff
@@ -191,11 +191,10 @@ private void setThreads(final @NotNull SentryEvent event) {
       // crashed properly
       List<Long> mechanismThreadIds = null;
 
-      final boolean hasExceptions =
-          event.getExceptions() != null && !event.getExceptions().isEmpty();
+      final List<SentryException> eventExceptions = event.getExceptions();
```

Comment:
wondering why we've changed the `hasExceptions` instead of checking NPE and emptiness 2 times

Reply:
`NullAway` is not able to recognize this check. If we want to keep using it, we would either have to do it as in PR, or disable warning for this method.